# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../glade/gtk214.patch
# Copyright (C) 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur glade-2.12.2-orig/glade/gbwidgets/gbclist.c glade-2.12.2/glade/gbwidgets/gbclist.c
--- glade-2.12.2-orig/glade/gbwidgets/gbclist.c	2009-02-15 14:00:30.000000000 +0000
+++ glade-2.12.2/glade/gbwidgets/gbclist.c	2009-02-15 14:02:08.000000000 +0000
@@ -19,10 +19,7 @@
 
 #include <string.h>
 
-#include <gtk/gtkclist.h>
-#include <gtk/gtkhbox.h>
-#include <gtk/gtkmain.h>
-#include <gtk/gtkspinbutton.h>
+#include <gtk/gtk.h>
 #include "../gb.h"
 
 /* Include the 21x21 icon pixmap for this widget, to be used in the palette */
diff -Nur glade-2.12.2-orig/glade/glade_keys_dialog.c glade-2.12.2/glade/glade_keys_dialog.c
--- glade-2.12.2-orig/glade/glade_keys_dialog.c	2009-02-15 14:00:30.000000000 +0000
+++ glade-2.12.2/glade/glade_keys_dialog.c	2009-02-15 14:03:01.000000000 +0000
@@ -23,11 +23,7 @@
 
 #include <string.h>
 
-#include <gtk/gtkbox.h>
-#include <gtk/gtkbutton.h>
-#include <gtk/gtkclist.h>
-#include <gtk/gtkstock.h>
-#include <gtk/gtkscrolledwindow.h>
+#include <gtk/gtk.h>
 
 #include "gladeconfig.h"
 
diff -Nur glade-2.12.2-orig/glade/glade_menu_editor.c glade-2.12.2/glade/glade_menu_editor.c
--- glade-2.12.2-orig/glade/glade_menu_editor.c	2009-02-15 14:00:30.000000000 +0000
+++ glade-2.12.2/glade/glade_menu_editor.c	2009-02-15 14:03:56.000000000 +0000
@@ -25,33 +25,7 @@
 #include <string.h>
 #include <time.h>
 
-#include <gdk/gdkkeysyms.h>
-#include <gtk/gtkarrow.h>
-#include <gtk/gtkaccellabel.h>
-#include <gtk/gtkclist.h>
-#include <gtk/gtkcombo.h>
-#include <gtk/gtkentry.h>
-#include <gtk/gtkeventbox.h>
-#include <gtk/gtkfilechooserdialog.h>
-#include <gtk/gtkframe.h>
-#include <gtk/gtkhbox.h>
-#include <gtk/gtkhbbox.h>
-#include <gtk/gtkhseparator.h>
-#include <gtk/gtkiconfactory.h>
-#include <gtk/gtkimage.h>
-#include <gtk/gtkimagemenuitem.h>
-#include <gtk/gtklist.h>
-#include <gtk/gtkmain.h>
-#include <gtk/gtkmenu.h>
-#include <gtk/gtkmenubar.h>
-#include <gtk/gtkradiobutton.h>
-#include <gtk/gtkradiomenuitem.h>
-#include <gtk/gtkscrolledwindow.h>
-#include <gtk/gtkseparatormenuitem.h>
-#include <gtk/gtkstock.h>
-#include <gtk/gtktable.h>
-#include <gtk/gtktearoffmenuitem.h>
-#include <gtk/gtkvbox.h>
+#include <gtk/gtk.h>
 
 #include "gladeconfig.h"
 
diff -Nur glade-2.12.2-orig/glade/glade_project_view.h glade-2.12.2/glade/glade_project_view.h
--- glade-2.12.2-orig/glade/glade_project_view.h	2009-02-15 14:00:30.000000000 +0000
+++ glade-2.12.2/glade/glade_project_view.h	2009-02-15 14:05:02.000000000 +0000
@@ -18,7 +18,7 @@
 #ifndef GLADE_PROJECT_VIEW_H
 #define GLADE_PROJECT_VIEW_H
 
-#include <gtk/gtkclist.h>
+#include <gtk/gtk.h>
 
 #include "glade_project.h"
 
